Kitodo/src/main/webapp/WEB-INF/templates/includes/massImport/textMassImport.xhtml

```diff
@@ -23,11 +23,11 @@
                     <p:outputLabel for="catalogueSelect"
                                    value="#{msgs['newProcess.catalogueSearch.catalogue']}"/>
                     <p:selectOneMenu id="catalogueSelect"
-                                     immediate="true"
```

Comment:
Same problem as above, but in this case in addition to the `ConfigException` in the backend, the user only sees a white page if he didn't select a valid OPAC:
```
org.kitodo.exceptions.ConfigException: The catalog '' is not found!
	at org.kitodo.config.OPACConfig.getCatalog(OPACConfig.java:226)
```
Additionally, the `Mass import` button here should also be deactivated if the text input field is empty.